scripts/docker/README.md

```diff
@@ -0,0 +1,63 @@
+## Docker Image
+
+Included in this repo is a Dockerfile that you can launch harmony node for trying it out. Docker images are available on `harmonyone/harmony`.
+
+You can build the docker image with the following commands:
+```bash
+make docker
+```
+
+If your build machine has an ARM-based chip, like Apple silicon (M1), the image is built for `linux/arm64` by default. To build for `x86_64`, apply the --platform arg:
+
+```bash
+docker build --platform linux/amd64 -t harmonyone/harmony -f Dockerfile .
+```
+
+Before start the docker, dump the default config `harmony.conf`, make your customization. `harmony.conf`  should be mounted into default `HOME` directory `/harmony` inside the container. Assume `harmony.conf`, `blskeys` and `hmykey` are under `./config` in your current working directory, you can start your docker container with the following command:
```

Comment:
Can you add a one liner instruction that shows how to dump the `harmony.conf` out from the docker image `harmonyone/harmony` ?

Reply:
done